### DISCUSSION TITLE
case-studies/women-of-resilience/

### DISCUSSION THREAD

**Discussion by giscus** [CommandPost/FCPCafe#336] [Website Discussions]
# case-studies/women-of-resilience/

CASE STUDY: BROADCAST MASTERING AND VERSIONING

https://fcp.cafe/case-studies/women-of-resilience/

**Comment by dustying** [comment]
Phenomenal write up, thank you! I love seeing how professionals are relying on these tools. 
Can't wait to see the film, that looks great too!
